articles/language/statements.md

```diff
@@ -273,6 +273,36 @@ sets the second element of the `result` array to `One`.
 Note that the `result` array must have been defined in a `mutable` statement
 for this to be valid.
 
+
+..............
```

Comment:
What's this line for? It doesn't seem to render right at the moment.

Reply:
this was the file that I was in the middle of editing - will be removed with the PR beheim/release0.6_part2 going into this branch